index.html

```diff
@@ -54,12 +54,14 @@ <h1 class="headings">ABOUT ME</h1>
     </section>
 
     <section id="portfolio">
-      <h1 class="alt-headings">PORTFOLIO</h1>
+      <h1 class="headings-first">PORTFOLIO</h1>
+      <img
+        src="https://user-images.githubusercontent.com/24613646/88513122-916e6380-cf9c-11ea-89e4-c93b628fc830.png"
+        alt="refactor project"
+        width="600"
+        height="650"
+      />
```

Comment:
try and avoid doing inline styling like this. This could go in your CSS file.

Reply:
This should also have a link to the deployed project as mentioned on your feedback.